MIGRATION_GUIDE.md

```diff
@@ -90,6 +90,30 @@ Some of the resources are excluded from this change:
 #### *(breaking change)* removed `qualified_name` from `snowflake_masking_policy`, `snowflake_network_rule`, `snowflake_password_policy` and `snowflake_table`
 Because of introducing a new `fully_qualified_name` field for all of the resources, `qualified_name` was removed from `snowflake_masking_policy`, `snowflake_network_rule`,  `snowflake_password_policy` and `snowflake_table`. Please adjust your configurations. State is automatically migrated.
 
+### snowflake_stage resource changes
+
+#### *(bugfix)* Correctly handle renamed/deleted stage
+
+Correctly handle the situation when stage was rename/deleted externally (earlier it resulted in a permanent loop). No action is required on the user's side.
```

Comment:
Hmm, I'm not sure if a bugfix that changes nothing on the user side should belong to a migration guide.